decidim-core/app/views/layouts/decidim/_head.html.erb

```diff
@@ -22,3 +22,4 @@
 <%= javascript_include_tag 'application' %>
 
 <%= render partial: 'layouts/decidim/head_extra' %>
+<%== current_organization.header_snippets %>
```

Comment:
Not 100% about this, but I believe `text` columns don't get loaded with the initial database select, but asked again afterwards if needed (to prevent huge amounts of memory from being used). 

If this is the case, I think something like `Oraganization.preload(:header_snippets)` would work.

Reply:
`preload` is used for relations:

https://github.com/rails/rails/blob/6cf4835b743cbce2536d53ae0a4a998f4f0f1e24/activerecord/lib/active_record/relation/query_methods.rb#L170

Checking the logs, I only see one hit on the database to get the organization, other hits are cached, so I guess it's working OK as it is.